pilot/pkg/kube/inject/monitoring.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package inject
+
+import (
+	"istio.io/istio/pilot/pkg/monitoring"
+)
+
+var (
+	totalInjections = monitoring.NewSum(
+		"sidecar_total_injection_requests",
```

Comment:
nit: Seems most metrics are following a pattern of `x_total` rather than `total_x`.

I am not sure if this is just a stylistic thing or if there is a real reason, but may be best to be consistent

Reply:
There is a best practices for this: https://prometheus.io/docs/practices/naming/